__tests__/integration/ng-list-no-reload.test.tsx

```
@@ -0,0 +1,78 @@
+  let originalLocation: Location
+  let reloadMock: vi.Mock
+
+  beforeEach(() => {
+    // Save original location
+    originalLocation = window.location
+    
+    // Create a mock for window.location.reload
+    reloadMock = vi.fn()
+    
+    // Mock window.location
+    delete (window as any).location
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)